content/tutorials/queue-stomp-neutrino-profanity.textile

```diff
@@ -233,7 +233,7 @@ We're now ready to start consuming messages from the queue. There are "plenty of
 
 ```[json]
   "dependencies": {
-    "ably": ">=0.9.0-beta",
+    "ably": "~1.0",
```

Comment:
> `"~1.0"`

I don't think this is right, assuming we want something that matches `>= 1.0.0 && < 2.0.0` -- you want either `~1` or `^1.0`; `~1.0` locks into the minor version

(similarly for the others)